plugins/statistics.koplugin/main.lua

```diff
@@ -33,7 +34,7 @@ local DEFAULT_CALENDAR_START_DAY_OF_WEEK = 2 -- Monday
 local DEFAULT_CALENDAR_NB_BOOK_SPANS = 3
 
 -- Current DB schema version
-local DB_SCHEMA_VERSION = 20201022
+local DB_SCHEMA_VERSION = 20221022
```

Comment:
Dunno if you can update this date to today - or if it will mess with your own tests dbs on your various devices.

Reply:
I can manually update my dbs. Today as 1111 is also more preferable for me ;)